.github/workflows/commits.yml

```diff
@@ -0,0 +1,33 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+name: Conventional commit check
+on: [push]
+jobs:
+  commit_messages:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Load scopes from file
+        id: load_scopes
+        run: |
+          scopes_file=".github/scopes.txt"
+          scopes=$(awk '{print $1}' $scopes_file)
+          echo "::set-output name=scopes::$scopes"
+
+      - name: Conventional commit check
+        id: conventional_commit_check
+        run: |
+          regex="^(build|feat|fix|test|perf|refactor|docs|style)(\(($scopes)\))?:\s.*(\n\n.*)*(\n\n(\w|[-/])+.*)*$"
```

Comment:
Move commit types to file too?

Reply:
This is standard from conventional commit, it won't change or update no matter what we do in the project. I don't think we need to keep it separate, only if we want to keep this command cleaner

Reply:
Ok, fair enough.